astro/src/content/docs/apis/_tenant-request-body.mdx

```diff
@@ -921,6 +921,32 @@ import TransactionTypes from 'src/content/docs/apis/_transaction-types.mdx';
 
   </APIField>
 
+  <APIField name="tenant.rateLimitConfiguration.sendPasswordlessPhone.enabled" type="Boolean" optional defaults="false" since="1.99.9">
+    Whether rate limiting is enabled for send passwordless via phone.
+
+    <EnterprisePlanBlurbApi feature="rate limiting" />
+
+  </APIField>
+
+  <APIField name="tenant.rateLimitConfiguration.sendPasswordlessPhone.limit" type="Integer" optional defaults="5" since="1.99.9">
+    The number of times a user can request a passwordless login message within the configured <InlineField>timePeriodInSeconds</InlineField> duration.
+
+    Required when `enabled` is set to `true`.
+
+    <EnterprisePlanBlurbApi feature="rate limiting" />
+
+  </APIField>
+
+  <APIField name="tenant.rateLimitConfiguration.sendPasswordlessPhone.timePeriodInSeconds" type="Integer" optional defaults="60" since="1.99.9">
+    The duration for the number of times a user can request a passwordless login message before being rate limited.
```

Comment:
```suggestion
    The duration for the number of times a user can request a passwordless login SMS message before being rate limited.
```

Reply:
I can't make up my mind about how much to talk about SMS or not